packages/eslint-config/.eslintrc.js

```diff
@@ -0,0 +1,66 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
```

Comment:
I believe the convention is to call this file `eslint.json`, see e.g. https://github.com/strongloop/eslint-config-loopback/blob/master/eslint.json

I am ok to use JavaScript too, in which case I think the file should be called `eslint.js`.